src/cli/cli_history.cpp

```diff
@@ -134,6 +182,54 @@ template <> otError History::Process<Cmd("ipaddr")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli history ipmaddr
+ * @code
+ * history ipmaddr
+ * | Age                  | Event        | Multicast Address                       | Origin |
+ * +----------------------+--------------+-----------------------------------------+--------+
+ * |         00:00:08.592 | Unsubscribed | ff05:0:0:0:0:0:0:1                      | Manual |
+ * |         00:01:25.353 | Subscribed   | ff05:0:0:0:0:0:0:1                      | Manual |
+ * |         00:01:54.953 | Subscribed   | ff03:0:0:0:0:0:0:2                      | Thread |
+ * |         00:01:54.953 | Subscribed   | ff02:0:0:0:0:0:0:2                      | Thread |
+ * |         00:01:59.329 | Subscribed   | ff33:40:fdde:ad00:beef:0:0:1            | Thread |
+ * |         00:01:59.329 | Subscribed   | ff32:40:fdde:ad00:beef:0:0:1            | Thread |
+ * |         00:02:01.129 | Subscribed   | ff03:0:0:0:0:0:0:fc                     | Thread |
+ * |         00:02:01.129 | Subscribed   | ff03:0:0:0:0:0:0:1                      | Thread |
+ * |         00:02:01.129 | Subscribed   | ff02:0:0:0:0:0:0:1                      | Thread |
+ * Done
+ * @endcode
+ * @code
+ * history ipmaddr list
+ * 00:00:25.447 -> event:Unsubscribed address:ff05:0:0:0:0:0:0:1 origin:Manual
+ * 00:01:42.208 -> event:Subscribed address:ff05:0:0:0:0:0:0:1 origin:Manual
+ * 00:02:11.808 -> event:Subscribed address:ff03:0:0:0:0:0:0:2 origin:Thread
+ * 00:02:11.808 -> event:Subscribed address:ff02:0:0:0:0:0:0:2 origin:Thread
+ * 00:02:16.184 -> event:Subscribed address:ff33:40:fdde:ad00:beef:0:0:1 origin:Thread
+ * 00:02:16.184 -> event:Subscribed address:ff32:40:fdde:ad00:beef:0:0:1 origin:Thread
+ * 00:02:17.984 -> event:Subscribed address:ff03:0:0:0:0:0:0:fc origin:Thread
+ * 00:02:17.984 -> event:Subscribed address:ff03:0:0:0:0:0:0:1 origin:Thread
+ * 00:02:17.984 -> event:Subscribed address:ff02:0:0:0:0:0:0:1 origin:Thread
+ * Done
+ * @endcode
+ * @cparam history ipmaddr [@ca{list}] [@ca{num-entries}]
+ * * Use the `list` option to display the output in list format. Otherwise,
+ *   the output is shown in table format.
+ * * Use the `num-entries` option to limit the output to the number of
+ *   most-recent entries specified. If this option is not used, all stored
+ *   entries are shown in the output.
+ * @par
+ * Displays the multicast IPv6 address history in table or list format.
+ * @par
+ * Each table or list entry provides:
+ * * Age: Time elapsed since the command was issued, and given in the format:
+ *        `hours`:`minutes`:`seconds`:`milliseconds`
+ * * Event: Possible values are `Subscribed` or `Unsubscribed`.
+ * * Multicast Address
+ * * Origin: Possible values are `Thread` or `Manual`.
+ * @sa otHistoryTrackerEntryAgeToString
```

Comment:
Related... can you figure out why the content for this function doesn't output correctly on openthread.io?  

https://openthread.io/reference/group/api-history-tracker#othistorytrackerentryagetostring

Needs a fix here on GitHub